packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_inputs.dart

```diff
@@ -400,4 +399,16 @@ mixin _PropertyInputMixin<T extends StatefulWidget, U> on State<T> {
       ),
     );
   }
+
+  String _errorMessage(
+    EditArgumentResponse errorResponse, {
+    required EditableProperty property,
+  }) {
+    final errorType = errorResponse.errorType;
+    final errorMessage =
+        errorType?.message ?? errorResponse.errorMessage ?? 'Unknown error.';
```

Comment:
Is it ever the case that both `errorType.message` and `errorResponse.errorMessage` might have useful info?

Reply:
Good point! Now including both (if both exist)